add entry for deprecation re: new company synchronized webhook (#460)
codatio/codat-docs: blog/231009-deprecation-webhooks-new-company-synchronized.md

---

> title: "Upcoming 2023-10-09: Webhooks - 'New company synchronized' alert triggers after all datasets are complete"

Reviewer: ```suggestion
title: "Upcoming 2023-10-09: Webhooks - 'New company synchronized' notification triggers after all datasets are complete"
```

> title: "Upcoming 2023-10-09: Webhooks - 'New company synchronized' alert triggers after all datasets are complete"

Reviewer: we no longer use the word "alert" when talking about webhooks 🙃 it's "events" for raised alerts, and usually notifications or webhooks for other scenarios.

date: "2023-06-13"
tags: ["Deprecation"]
authors: mrenshawcodat
---

> On October 9, 2023, alerts for the 'New company synchronized' webhook will trigger after all datasets created during the initial sync of a company have completed.

Reviewer: ```suggestion
From October 9, 2023, notifications for the 'New company synchronized' webhook will only trigger after all datasets created during the initial sync of a company are completed.
```

<!--truncate-->

> Currently, this alert will trigger after the first dataset queued as a result of linking a company is complete and successful. 

Reviewer: ```suggestion
Currently, this notification triggers after the first dataset queued when linking a company is successfully synced. 
```

After October 9, 2023, the alert will only trigger when all datasets queued as a result of linking a company are complete, and at least one of those datasets is successful. 

## Action required​

This is a quality of life improvement that means you will be able to rely on this alert to know when a company is no longer in the process of syncing data. 

> If you have workflows that are dependent on the alert firing after only the first dataset is complete, then these will need to be adjusted to expect the alert to be received after all datasets are complete.

Reviewer: ```suggestion
If you have workflows that are dependent on the event of the first dataset completion, you will need to adjust these to expect the notification only after all datasets are complete.
```
